### ISSUE TITLE
How to configure PiHole on Syncloud?

### ISSUE THREAD

**Issue by Penguinjumper** [syncloud/platform#563]
So for me the Web Interface of the pihole app has many settings missing, for example importing lists.
I wanted to configure it myself via ssh (taught myself a few basics which are hopefully enough) but when typing 
pihole -h 
for example, I always get

-bash: pihole: command not found

Why is the pihole syncloud app not as configurable as vanilla pihole?
And how do I start the executable running the pihole commands?

Thanks! Otherwise great work, helping non-professionals getting this stuff set up.

**Comment by cyberb** [comment]
Unfortunately this app is based on outdated version, I have just created an issue to migrate to the latest one.
https://github.com/syncloud/platform/issues/564

For now we have this script to update ad lists which I think internally was used by pihole script:

 /snap/pihole/current/bin/gravity.sh

**Comment by Penguinjumper** [comment]
Thank you for the quick response, I guess I'll wait for the update then :)

**Comment by cyberb** [comment]
ok, follow the corresponding issue.

**Comment by Penguinjumper** [comment]
Maybe as a temporary workaround: I tried installing pihole via ssh and "curl -sSL https://install.pi-hole.net | bash"

For pihole, this worked, but it seems this broke something within syncloud, as I couldn't reach it's web ui or my nextcloud anymore.

(Since I'm a rookie with this stuff, I had to 'factory reset' the whole thing, because I really needed my nextcloud up and running.)

Anything I overlooked? Or is it just not possible to install other applications next to the "official" syncloud ones?
